kll/test/kll_sketch_test.cpp

```diff
@@ -115,7 +114,7 @@ TEST_CASE("kll sketch", "[kll_sketch]") {
     sketch.update(std::numeric_limits<float>::quiet_NaN());
     REQUIRE(sketch.is_empty());
 
-    sketch.update(0.0);
+    sketch.update(0);
```

Comment:
I vaguely remember that this generated a warning about conversion from double to float. perhaps this could be 0.0f, but just 0 works fine with no warning.